provd/servers/tftp/connection.py

```diff
@@ -2,13 +2,27 @@
 # SPDX-License-Identifier: GPL-3.0-or-later
 
 """Manage the transfer between two host."""
-
+from __future__ import annotations
 
 # TODO RFC1122 says we must use an adaptive timeout...
 
 import struct
 import logging
-from provd.servers.tftp.packet import *
+from typing import BinaryIO
+
+from provd.servers.tftp.packet import (
+    build_dgram,
+    err_packet,
+    ERR_UNKNWN_TID,
+    ERR_UNDEF,
+    ERR_ILL,
+    parse_dgram,
+    PacketError,
+    OP_ERR,
+    OP_ACK,
+    data_packet,
+    AckPacket,
```

Comment:
Do you know if it would be possible to configure the tool to sort these alphabetically?

Reply:
Yes, and I would love to do that. It would be as simple as including `isort` in pre-commit. I included it in my pre-commit proposition last year ([WAZO-2984](https://wazo-dev.atlassian.net/browse/WAZO-2984)). However, the best I could get accepted was to do it in two steps, and isort is in the second step. And for some reason the second step is conditional on adding pre-commit every repo first.... and after almost a year that still hasn't happened. 😢

Reply:
Yeah... it's sad. I'm trying to do them when I have time during freezes. But haven't made much progress.
Awesome, thanks! 
That's great!